.circleci/config.yml

```diff
@@ -20,7 +20,6 @@ jobs:
           paths:
             - .venv
       - run: pipenv run ruff check .
-      - run: pipenv run black --target-version py38 --check --diff .
```

Comment:
This should be `ruff format --check .` since all linting is happening in CircleCI